### PR TITLE
Scoring locatins in field

### PR DESCRIPTION
**Pull request by greenblitz4590**

### CODE REVIEW

**Review by aaa-png — APPROVED** [review]

File: src/main/java/edu/greenblitz/robotName/Field.java
Comment (yonnbro) [inline src/main/java/edu/greenblitz/robotName/Field.java:1]: let's create consts for real field and upstairs field

File: src/main/java/edu/greenblitz/robotName/commands/GetNoteToSystem/GetToTransferNotePosition.java
Comment (yonnbro) [inline src/main/java/edu/greenblitz/robotName/commands/GetNoteToSystem/GetToTransferNotePosition.java:1]: not relevant to this PR. if so - move to right PR

File: src/main/java/edu/greenblitz/robotName/commands/GetNoteToSystem/MoveShooterToAngle.java
Comment (yonnbro) [inline src/main/java/edu/greenblitz/robotName/commands/GetNoteToSystem/MoveShooterToAngle.java:1]: not relevant to this PR. if so - move to right PR

**Review by yonnbro — CHANGES_REQUESTED** [review]

File: src/main/java/edu/greenblitz/robotName/Field.java [hunk 1]
```diff
@@ -97,8 +97,8 @@ public static class FieldConstants{
     }
 
     public static class ScoringPositions{
-        public final static Pose2d SPEAKER_SCORE_POSITION = new Pose2d(new Translation2d(5,4),new Rotation2d());
-        public final static Pose2d AMP_SCORE_POSITION = new Pose2d(new Translation2d(3,1),new Rotation2d());
-        public final static Pose2d CLIMB_POSITION = new Pose2d(new Translation2d(7,2),new Rotation2d());
+        public final static Pose2d SPEAKER_SCORE_POSITION = new Pose2d(new Translation2d(14.140,5.85),new Rotation2d());
```
Comment (NoamRosenberg08) [inline src/main/java/edu/greenblitz/robotName/Field.java:100]: shouldntwesaqve this as arotation2D?
if therearecoliisiton with MoveToPose add a functions for translations

**Review by NoamRosenberg08 — CHANGES_REQUESTED** [review]

**Review by NoamRosenberg08 — APPROVED** [review]

**Review by tnrz135 — APPROVED** [review]

File: src/main/java/edu/greenblitz/robotName/Field.java
Comment (tnrz135) [reply src/main/java/edu/greenblitz/robotName/Field.java:1]: They did it but forgot to resolve and now it is blocking the merge

**Pull request merged**